source/val/validate_scopes.cpp

```diff
@@ -105,33 +105,65 @@ spv_result_t ValidateExecutionScope(ValidationState_t& _,
       }
     }
```

Comment:
Unrelated to your change, but the Vulkan specific rules seem to extend much further.

Reply:
not sure what you are trying to get at?

this is the closing bracket for `if (_.context()->target_env != SPV_ENV_VULKAN_1_0) {`